statics/js/components/status.js

```diff
@@ -0,0 +1,44 @@
+/* jshint multistr: true */
+
+var StatusComponent = {
+
+  name: 'Status',
+
+  mixins: [apiMixin, notificationMixin],
+
+  template: '\
+    <div class="status statustab">\
+      <panel title="Status" :collapsed="false">\
```

Comment:
I would make it collapsible, and I would set "Services status" as title

Reply:
done